src/Castle.Facilities.AspNet.Mvc.Tests/AspNetMvcFacilityTestCase.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright 2004-2017 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Facilities.AspNet.Mvc.Tests
+{
+	using System.IO;
+	using System.Web;
+	using System.Web.Mvc;
+	using System.Web.Routing;
+
+	using Castle.Facilities.AspNet.Mvc;
+	using Castle.Facilities.AspNet.Mvc.Exceptions;
+	using Castle.MicroKernel.Registration;
+	using Castle.Windsor;
+
+	using NUnit.Framework;
+
+	[TestFixture]
+    public class AspNetMvcFacilityTestCase
```

Comment:
There are formatting problems in many of these unit test files.

Reply:
Hopefully addressed in: 40213c2

Man! this is a constant battle :)

Reply:
Have you got the EditorConfig VS extension installed?